src/uncontrast/pages/UploadNewPaper/ExperimentsSection/ConsciousnessMeasures.jsx

```diff
@@ -442,11 +445,24 @@ export default function ConsciousnessMeasures({
           </div>
         );
       })}
-      <AddFieldButton
-        initialValues={initialValues}
-        fieldValues={fieldValues}
-        setFieldValues={setFieldValues}
-      />
+      <div className="w-full flex justify-center">
+        <Tooltip
+          placement="top"
+          content={
+            !disableAddBttns
```

Comment:
why both here and above? seems the same

Reply:
oh it's just the wrong text maybe? referencing finding instead of Consciousness Measures

Reply:
:-1: